vertical-pod-autoscaler/enhancements/8818-in-place-only/README.md

```diff
@@ -0,0 +1,218 @@
+# AEP-8818: Non-Disruptive In-Place Updates in VPA
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+- [Goals](#goals)
+- [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+- [Design Details](#design-details)
+- [Test Plan](#test-plan)
+- [Upgrade / Downgrade Strategy](#upgrade--downgrade-strategy)
+  - [Upgrade](#upgrade)
+  - [Downgrade](#downgrade)
+- [Feature Enablement and Rollback](#feature-enablement-and-rollback)
+  - [How can this feature be enabled / disabled in a live cluster?](#how-can-this-feature-be-enabled--disabled-in-a-live-cluster)
+- [Kubernetes version compatibility](#kubernetes-version-compatibility)
+- [Implementation History](#implementation-history)
+<!-- /toc -->
+
+## Summary
+
+[AEP-4016](https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler/enhancements/4016-in-place-updates-support) introduced the `InPlaceOrRecreate` update mode which attempts in-place updates first but falls back to pod eviction if the in-place update fails. However, for certain workloads, any disruption is unacceptable, and users would prefer to retry in-place updates indefinitely rather than evict and recreate pods.
+
+This proposal introduces a new update mode that only attempts in-place updates and retries on failure without ever falling back to eviction.
+
+## Motivation
+
+There are several use cases where pod disruption should be avoided at all costs:
+
+- Stateful workloads: Pods managing critical state where restart would cause data loss or lengthy recovery.
+- Long-running computations: Jobs or services performing computations that cannot be checkpointed and would need to restart from the beginning.
+- Strict SLO requirements: Services with stringent availability requirements where even brief disruptions are unacceptable.
+In these scenarios, users would prefer:
+
+- To operate with current (potentially suboptimal) resource allocations until an in-place update becomes feasible
+- To receive clear signals when updates cannot be applied
+- To have VPA continuously retry updates as cluster conditions change
+
+## Goals
+
+- Provide a truly non-disruptive VPA update mode that never evicts pods
+- Allow VPA to eventually apply updates when cluster conditions improve
+- Respect the existing in-place update infrastructure from AEP-4016
+
+## Non-Goals
+
+- Guarantee that all updates will eventually succeed (node capacity constraints may prevent this)
+- Provide mechanisms to automatically increase node capacity to accommodate updates
+- Change the behavior of existing update modes (Auto, Recreate, InPlaceOrRecreate)
```

Comment:
```suggestion
- Change the behavior of existing update modes (Off, Recreate, InPlaceOrRecreate)
```

Not sure if adding Auto makes sense, since it's deprecated? I'll let you decide.